doc/TSCC.rst

```diff
@@ -59,6 +67,10 @@ Communal lab resources are in :code:`/tscc/projects/ps-gymreklab/resources/`. Fe
 * :code:`/tscc/projects/ps-gymreklab/resources/dbase` contains reference genome builds for humans and mice and other
   non-project-specific datasets
 * :code:`/tscc/projects/ps-gymreklab/resources/datasets` contains project-specific datasets that are shared across the lab.
+* :code:`/tscc/projects/ps-gymreklab/resources/datasets/ukbiobank` contains our local copy of the UK Biobank. You must have the proper Unix permissions to read these files. Ask Melissa to add you on the UK Biobank portal and then ask to be added to the :code:`gymreklab-ukb` group afterwards.
+* :code:`/tscc/projects/ps-gymreklab/resources/datasets/1000Genomes` contains files for the 1000 Genomes dataset
+* :code:`/tscc/projects/ps-gymreklab/resources/datasets/gtex` contains the GTEX dataset
+* :code:`/tscc/projects/ps-gymreklab/resources/datasets/pangenome` contains pangenome files
 
```

Comment:
```suggestion

Access TSCC files locally on your computer
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
You can upload and download files from TSCC using the `scp` command. Assuming you've configured a host in your `~/.ssh/config` named `tscc`, you would download chrM of the hg19 reference genome like this, for example.

.. code-block:: bash

  scp -r tscc:/tscc/projects/ps-gymreklab/resources/dbase/human_by_chrom/hg19/chrM.fa .

However, if you would like to download many files from TSCC or edit files on TSCC in real time, you may opt to mount TSCC as a network drive, instead. A program called `sshfs` will allow you to view and edit TSCC files on your computer and keep them synced with TSCC.

To set up `sshfs`, you must first download and install it. With `homebrew`, you can just do `brew install sshfs` or on Ubuntu or WSL, you can do `sudo apt install sshfs`. Next, simply add the following snippet to your `~/.bashrc`:

.. code-block::bash

    # mount a remote drive over ssh
    # arg 1: the hostname of the server, as specified in your ssh config
    # arg 2 (optional): the mount directory; defaults to arg1 in the current directory
    sshopen() {
        # perform validation checks, first
        command -v sshfs >/dev/null 2>&1 || { echo >&2 "error: sshfs is not installed"; return 1; }
        grep -q '^user_allow_other' /etc/fuse.conf || { echo >&2 "error: please uncomment the 'user_allow_other' option in /etc/fuse.conf"; return 1; }
        ssh -q "$1" exit >/dev/null || { echo >&2 "error: cannot connect to '$1' via ssh; check that '$1' is in your ~/.ssh/config"; return 1; }
        [ -d "${2:-$1}" ] && { ls -1qA "${2:-$1}" | grep -q .; } >/dev/null 2>&1 && { echo >&2 "error: '${2:-$1}' is not an empty directory; is it already mounted?"; return 1; }
        # set up a trap to exit the mount before attempting to create it
        trap "cd \"$PWD\" && { fusermount -u \"${2:-$1}\"; rmdir \"${2:-$1}\"; }" EXIT && mkdir -p "${2:-$1}" && {
            # ServerAlive settings prevent the ssh connection from dying unexpectedly
            # cache_timeout controls the number of seconds before sshfs retrieves new files from the server
            sshfs -o allow_other,reconnect,ServerAliveInterval=15,ServerAliveCountMax=3,cache_timeout=900,follow_symlinks "$1": "${2:-$1}"
        } || {
            # if the sshfs command didn't work, store the exit code, clean up the dir and the trap, and then return the exit code
            local exit_code=$?
            rmdir "${2:-$1}" && trap - EXIT
            return $exit_code
        }
    }

After sourcing your `~/.bashrc` you should now be able to run :code:`sshopen tscc`! This will create a folder on your computer with all of the files from your home directory on TSCC. The network mount will be disconnected when you close your terminal.

Some notes on usage:
- Depending on your network connection, sshopen might choke on large files. Consider using scp for such files, instead.
- In order to reduce network usage, sshopen will only retrieve new files from the server every 15 minutes. If you want this to happen more frequently, just change the cache_timeout setting in the sshfs command.
- The unmount will fail if any processes are still utilizing files in the mount, so you should close Windows File Explorer and any other applications before you close your terminal window. If the unmount fails, you can always unmount manually: run :code:`pkill ssh` to kill the `sshfs` command and :code:`rmdir tscc` to delete the mounted folder.
```